tools/code_format/config.yaml

```diff
@@ -231,6 +231,7 @@ paths:
     include:
     - source/common/common/thread.h
     - source/exe/terminate_handler.cc
+    - source/common/json/json_sanitizer.cc
```

Comment:
let's not do this; this is really a violation and we shouldn't just ignore it.

Reply:
I think we back to the previous problem: how to handle the exception.